.clang-format

```diff
@@ -34,27 +34,25 @@ IncludeCategories:
     Priority:        4
   - Regex:           '^<flann/'
     Priority:        5
-  - Regex:           '^<(Q|q)[^/]+>'
+  - Regex:           '^<Q[^/]+>'
     Priority:        6
-    SortPriority:    6
   - Regex:           '^<ui_[^/]+\.h>$'
     Priority:        6
-    SortPriority:    7
   - Regex:           '^<vtk[^/]+\.h>$'
-    Priority:        8
+    Priority:        7
   - Regex:           '^<librealsense2/'
-    Priority:        9
+    Priority:        6
   - Regex:           '^<(ros|message_filters)/'
-    Priority:        10
+    Priority:        9
   - Regex:           '^<opencv(2)?/'
-    Priority:        11
+    Priority:        10
   - Regex:           '^<tide/'
-    Priority:        12
+    Priority:        11
   - Regex:           '^<thrust/'
-    Priority:        13
+    Priority:        12
   - Regex:           '^<(OpenGL|(GL(UT)?/))'
-    Priority:        14
+    Priority:        13
   - Regex:           '^<[a-z]+>$'
-    Priority:        16
-  - Regex:           '.*'
     Priority:        15
+  - Regex:           '.*'
+    Priority:        14
```

Comment:
You missed a '\n' at the end of the file.

Also, can we add some comment for the last 2 regex (others are pretty straightforward for even those who know a little regex)

Reply:
Comments added 👍